dev-db/postgrest-bin/postgrest-bin-0.4.4.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user eutils systemd
+
+MY_PN="${PN/-bin/}"
+
+DESCRIPTION="PostgREST serves a fully RESTful API from existing PostgreSQL database (Binary)"
+HOMEPAGE="https://postgrest.com"
+
+uri() {
+	echo "https://github.com/begriffs/postgrest/releases/download/v${PV}/${MY_PN}-v${PV}-ubuntu$1.tar.xz"
+}
+
+SRC_URI="
+		x86? ( $(uri i386) )
+		amd64? ( $(uri) )
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	!dev-db/postgrest
+	dev-db/postgresql"
+
+QA_PREBUILT="/usr/bin/postgrest-bin"
+QA_PRESTRIPPED="/usr/bin/postgrest-bin"
+
+S=${WORKDIR}
+
+USER_NAME="postgrest"
```

Comment:
there is no need to introduce one more variable, pass it to the `pkg_setup()` directly